romsel_aktheme/nitrofiles/themes/zelda/uisettings.ini

```diff
@@ -25,6 +25,7 @@ y = 172
 w = 48
 h = 18
 textColor = 0x7FFF
+text = ini
```

Comment:
what does this do?

Reply:
Read START text from language.txt file like the orginal wood.